contract-message-predicate/src/contract_message_test.sw

```diff
@@ -41,25 +41,26 @@ impl MessageReceiver for Contract {
     #[storage(read, write)]
     #[payable]
     fn process_message(msg_idx: u8) {
-        storage.counter = storage.counter + 1;
+        storage.counter.write(0); // Temporary fix for: https://github.com/FuelLabs/sway/issues/4634
```

Comment:
Hmm, we need to do that during initialization, otherwise, the counter is always 1=)